CHANGELOG.md

```diff
@@ -9,6 +9,7 @@ Inspired from [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 - Added overload decorators to helpers-actions.pyi-"bulk" ([#239](https://github.com/opensearch-project/opensearch-py/pull/239))
 - Document Keberos authenticaion ([214](https://github.com/opensearch-project/opensearch-py/pull/214))
 - Add release workflows ([#240](https://github.com/opensearch-project/opensearch-py/pull/240))
+- Added new OpenSearch versions and updated compatibility matrix ([#257](https://github.com/opensearch-project/opensearch-py/pull/257))
```

Comment:
We should be explicit here and say something like "Compatibility with OpenSearch 2.1.0 - 2.4.1".